### ISSUE TITLE
up logs sort does not factor in am / pm

### ISSUE THREAD

**Issue by sanjaypojo** [apex/up#715]
## Prerequisites

* [x ] I am running the latest version. (`up upgrade`)
* [x] I searched to see if the issue already exists.
* [x] I inspected the verbose debug output with the `-v, --verbose` flag.
* [x] Are you an Up Pro subscriber?

## Description
`up logs` prints logs sorted by time such that the `am` logs show up after the `pm` logs (should be the other way)

## Steps to Reproduce
Run `up logs` in the evening.

**Comment by tj** [comment]
oh weird! I think I'm just printing them as they're received from AWS, should be ascending but I'll see if I can reproduce

**Comment by tj** [comment]
looks correct in mine, possibly a race condition though I'll keep digging

**Comment by sanjaypojo** [comment]
Yeah, I didn't have this issue before, but it's happening to me now:
![image](https://user-images.githubusercontent.com/4957294/46502646-f5a0c000-c828-11e8-8c26-595085c2494c.png)
